src/ontology/mondo-edit.obo

```diff
@@ -517780,11 +517780,12 @@ is_a: MONDO:0002254 ! syndromic disease
 
 [Term]
 id: MONDO:0042605
-name: Y chromosome infertility due to DAZ1 deletion
+name: Y chromosome infertility
+def: "A condition that affects the production of sperm, making it difficult or impossible for affected men to father children." [GARD:0000185]
```

Comment:
this definition is too broad for the meaning of "Y chromosome infertility": other conditions can affect production of sperm

Reply:
ok, that was the description from GARD